metered/src/time_source.rs

```diff
@@ -8,20 +8,44 @@ pub trait Instant {
     fn now() -> Self;
 
     /// Returns the elapsed time in milliseconds since an Instant was created.
-    fn elapsed_millis(&self) -> u64;
+    fn elapsed_time(&self) -> u64;
```

Comment:
This comment would have to be updated to match the new semantics of the method,
e.g
```rust
/// Returns the elapsed time since an Instant was created.
///
/// The unit depends on the Instant's resolution, as defined by the `ONE_SEC` constant.
```